wire-ios/Wire-iOS Tests/ConversationMessageActionControllerTests.swift

```diff
@@ -177,4 +177,18 @@ final class ConversationMessageActionControllerTests: XCTestCase, CoreDataFixtur
         // THEN
         XCTAssertFalse(supportsForward)
     }
+
+    func testGivenURLMessageThenSupportsVisitLink() {
+        // GIVEN
+        let message = MockMessageFactory.linkMessage()
+        message!.senderUser = MockUserType.createUser(name: "Bob")
+        message!.conversation = otherUserConversation
+
+        // WHEN
+        let actionController = ConversationMessageActionController(responder: nil, message: message!, context: .content, view: UIView())
```

Comment:
check but I don't see why `MockMessageFactory.linkMessage()` returns an optional? so we can remove the force unwrap